lib/ansible/module_utils/network/nxos/argspec/lacp/lacp.py

```diff
@@ -0,0 +1,49 @@
+#
+# -*- coding: utf-8 -*-
+# Copyright 2019 Red Hat
+# GNU General Public License v3.0+
+# (see COPYING or https://www.gnu.org/licenses/gpl-3.0.txt)
+
+#############################################
+#                WARNING                    #
+#############################################
+#
+# This file is auto generated by the resource
+#   module builder playbook.
+#
+# Do not edit this file manually.
+#
+# Changes to this file will be over written
+#   by the resource module builder.
+#
+# Changes should be made in the model used to
+#   generate this file or in the resource module
+#   builder template.
+#
+#############################################
+
+"""
+The arg spec for the nxos_lacp module
+"""
+from __future__ import absolute_import, division, print_function
+__metaclass__ = type
+
+
+class LacpArgs(object):
+    """The arg spec for the nxos_lacp module
+    """
+
+    def __init__(self, **kwargs):
+        pass
+
+    argument_spec = {'config': {'options': {'system': {'options': {'mac': {'options': {'address': {'type': 'str'},
```

Comment:
Could we maybe sort this out so it's easier to see? I mean having something in the lines of
{'config':
    {'options':
        {'system':

And so. It'd be just so much more readable.